_docs/backenders/how-we-work/index.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+title: How we work
+parent: Backenders
+has_children: false
+has_toc: false
+---
+
+The sections below are an initial attempt at summarising in an hopefully
+understandable and meaningful way what should be overall common traits of the
+backend group’s way of working, so that we can aim to be more intentional about
+how to represent our group’s work to potential candidates, whether in job
+descriptions or during job interviews and any follow-up questions.
+
+The tone is intentionally sober (mostly) as the aim, which is part of what we
+identified as needs during the April 2020 turnover retrospective, is to try to
+be clear with candidates (and between ourselves) about what we do and how we
+work: we would like to “sell” what is good about working as part of the
+functional area and at Vizzuality, and we should do so in as a honest and clear
+way as possible to avoid misalignment of expectations.
+
+## Team collaboration
+
+In most projects of small to medium complexity and lasting from a few weeks to a
+few months, backend development is typically done by a single engineer.
+
+In more complex or longer projects, a second backend engineer may be part of the
+team throughout the project or for part of it. Occasionally we may also add
+freelancers depending on load and schedules.
+
+We have been exploring how to consistently allocate a "sidekick" backend
+engineer on most projects so that peaks in workload can be smoothed, knowledge
+can be shared more broadly and there isn't a single "point of failure", but we
+still need to make this work.
+
+Whether as sole backend engineers or with one or more team members, we work in
+very close contact with user researchers, designers, frontend engineers,
+scientist, data engineers and project managers, both through scheduled stand up
+meetings and spring planning and retrospectives and by proactively reaching out
+to plan ahead, discuss functional design, unblock other project members' work.
+
+We may occasionally pair up remotely to work together on specific functional
+design tasks, to develop features, triage and fix bugs and to seek feedback and
+advice, although we don't use pair programming as a daily way of working.
+
+We take code reviews both as an opportunity to improve our projects and to learn
+and mentor.
+
+We share interesting bits of our daily work at our weekly team meeting. We do
```

Comment:
Let's rewrite this to reflect two types of meetings. e.g. We meet twice a week for two distinct types of team meetings. The first one is an organisational one, dedicated to updating each other and making team decisions broadly related to running a self-managed organisation. The other one is a technical meeting, where we share interesting bits of our daily work. We do also share frustrations, celebrate achievements and in general to sustain the team's growth, while keeping a light-hearted spirit. Once a month that meeting is held together with the Front End engineers team.

Reply:
great, thanks for the text suggestion - I have used it verbatim ❤️